recipes/r-baseset/meta.yaml

```
@@ -0,0 +1,108 @@
+{% set version = "0.9.0" %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-baseset
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/BaseSet_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/BiocManager/BaseSet_{{ version }}.tar.gz
+  sha256: e5c6d1609ae818427e4af97516a0a879969165c43bf2f36271fc12b56ac7c436
+
+build:
+  merge_build_host: true  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+  host:
+    - r-base
+    - r-dplyr >=1.0.0
+    - r-magrittr
+    - r-rlang
+  run:
+    - r-base
+    - r-dplyr >=1.0.0
+    - r-magrittr
+    - r-rlang
+
+test:
+  commands:
+    - $R -e "library('BaseSet')"           # [not win]
+    - "\"%R%\" -e \"library('BaseSet')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=BaseSet
```

Defer to the upstream URLs found in DESCRIPTION:

```suggestion
  home: https://docs.ropensci.org/BaseSet/
  dev_url: https://github.com/ropensci/BaseSet
```